src/composables/UserAuthentication.js

```diff
@@ -1,78 +1,29 @@
 import * as api from 'src/composables/LetoModelizerApi';
 import { useUserStore } from 'src/stores/UserStore';
 
-/**
- * Set the current user's session token in the local storage.
- * @param {string} sessionToken - The session token to store.
- */
-export function setUserSessionToken(sessionToken) {
-  localStorage.setItem('sessionToken', sessionToken);
-}
-
-/**
- * Get the current user's session token from the local storage.
- * @returns {string|null} The user's session token from the local storage.
- */
-export function getUserSessionToken() {
-  return localStorage.getItem('sessionToken');
-}
-
-/**
- * Remove the current user's session token from the local storage.
- */
-export function removeUserSessionToken() {
-  localStorage.removeItem('sessionToken');
-}
-
-/**
- * Log in the user using its temporaryCode. Once the user is logged, it saves
- * the user information in the dedicated store.
- * @param {string} temporaryCode - The temporary code obtained from the github provider.
- * @returns {Promise<string>} Promise with the user's token on success
- * otherwise an error.
- */
-export async function login(temporaryCode) {
-  return api.login(temporaryCode)
-    .then((response) => {
-      const userStore = useUserStore();
-
-      setUserSessionToken(response.data.sessionToken);
-
-      userStore.id = response.data.objectId;
-      userStore.username = response.data.username;
-      userStore.firstname = response.data.firstname;
-
-      return response.data.sessionToken;
-    });
-}
-
 /**
  * Initialize (GET request) information about the current user and store it in the dedicated store.
- * @param {string} sessionToken - The current user's session token.
  * @returns {Promise<object>} Promise with the logged-in user information on success
  * otherwise an error.
  */
-export async function initUserInformation(sessionToken) {
-  return api.getUserInformation(sessionToken)
+export async function initUserInformation() {
+  return api.getUserInformation()
     .then((response) => {
       const userStore = useUserStore();
 
-      userStore.id = response.data.objectId;
-      userStore.username = response.data.username;
-      userStore.firstname = response.data.firstname;
+      userStore.username = response.data.login;
+      userStore.firstname = response.data.name;
     });
 }
 
 /**
- * Retrieve all roles of user and store them in the dedicated store.
- * @param {string} userId - User id.
- * @param {string} sessionToken - The current user's session token.
+ * Retrieve all permissions of user and store them in the dedicated store.
```

Comment:
suggestion
`Retrieve (GET request) all permissions of user and store them in the dedicated store.`

Reply:
done